### ISSUE TITLE
The "LIKES" animations multiply after several clicks

### ISSUE THREAD

**Issue by renanleonellocastro** [MicrosoftDocs/msteams-docs#1175]
I was using the desktop application of teams in my computer and I realize that if you click several times in different "LIKES" type in the same comment, the like type instead of replacing each other, it will multiply and a lot of "LIKES" type of the same person will start appearing in the same message you liked before. Below there is a sample of what happened after several clicks in the "LIKE" button.

![screen1](https://user-images.githubusercontent.com/10985679/69727897-25ccb180-1102-11ea-82eb-4cfbbfec934b.png)

![screen2](https://user-images.githubusercontent.com/10985679/69727910-2b29fc00-1102-11ea-8bb0-3cffefa31e43.png)

**Comment by Gousia-MSFT** [comment]
@renanleonellocastro Thanks for reaching out. We cannot reproduce the issue. Could you please use the report a problem feature to report this? Our team will then be able to look into this.
![image](https://user-images.githubusercontent.com/45553518/70307949-4696b480-1830-11ea-901a-d2bd137f8d32.png)

**Comment by renanleonellocastro** [comment]
In the help button, I am not able to see the "Report a problem" item.

![image](https://user-images.githubusercontent.com/10985679/70345411-c45fbd80-183a-11ea-9f58-508d994d51ba.png)

**Comment by Gousia-MSFT** [comment]
@renanleonellocastro Could you please share the logs captured during the repro? These logs can be obtained by pressing Ctrl+ALt+Shift+1 with teams app in focus.

**Comment by renanleonellocastro** [comment]
Yes sure. The logs are attached here. The log file is about the following 2 like types (heart and thumbs up) in the same message from the same person.
![image](https://user-images.githubusercontent.com/10985679/70838472-22fedb80-1de7-11ea-9c21-8abb9f2dea80.png)
![image](https://user-images.githubusercontent.com/10985679/70838578-bc2df200-1de7-11ea-94a7-b0ca296a0063.png)
[MSTeams Diagnostics Log 12_13_2019__8_24_24_PM.txt](https://github.com/MicrosoftDocs/msteams-docs/files/3963094/MSTeams.Diagnostics.Log.12_13_2019__8_24_24_PM.txt)

**Comment by Gousia-MSFT** [comment]
@renanleonellocastro Thanks for reporting this. We have looked into the issue and seems like this is fixed in internal rings. It should reach production builds soon.
